### DISCUSSION TITLE
Capitation Arrangements; How should partially capitated arrangements be notated?

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#10] [Q&A]
Within the In-Network Rate File, there are different capitation arrangements that can be reported.   The rule indicates that partially capitated arrangement should be 'notated'.   How should partial capitated arrangements be 'notated' in the In-Network Rate File?   Will there be other examples or guidance for this in the capitation examples?   
Any schema updates?   Assume negotiation_arrangement is still 'capitation' or will there be a 'partial capitation' value?  Will the type of capitation arrangement need to be denoted?

**Comment by shaselton-usds** [comment]
Would you be able to provide a little more detail? For the partial capitation, would the non-capitation portion be something other than a fee-for-service or bundle arrangement? 

For multiple arrangements in a single plan, you can add an additional object in the [In Network Object in the `in_network` array](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#in-network-object) to report of different types of arrangements.

For your example, would it be possible to cover a partial capitation with a fee-for-service component like this?:
```
{
  "in_network": [{ 
    "negotiation_arrangement": "capitation",
    "name": "Primary Care Capitation",
    "description": "Typical items and services for a primary care provider",
    "negotiated_rates": [{
      "negotiated_rate": 1400.00,
      "providers": [{
        "tin": "",
        "service_code": "01",
        "npi": ["6666666666", "7777777777", "8888888888", "9999999999"]
      },
    "covered_services": [{
      "billing_code_type": "CPT",
      "billing_code_type_version": "2020",
      "billing_code": "27447",
      "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee Joint"
    }]
  },{
    "negotiation_arrangement": "fee for service",
    "name": "Knee Replacement",
    "billing_code_type": "CPT",
    "billing_code_type_version": "2020",
    "billing_code": "27447",
    "description": "Arthroplasty, knee condyle and plateau, medial and lateral compartments",
    "negotiated_rates": [{
      "providers": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
      "tin": "11-1111111",
      "negotiated_price": {
        "negotiated_type": "negotiated",
        "negotiated_rate": 123.45,
        "expiration_date": "2022-01-01"
      }
    }]
```

**Comment by tdfow** [comment]
The rule included the reference indicating a notation for partial capitation, so I wondered if that indicated the negotiation arrangement would support a ‘partial capitation’ value or if it would remain as ‘capitation’….and a different notation.    From the rule, I thought it would be some type of value.
It would seem that for partial capitation you would have negotiated_rates and have covered_services for each item/service that was part of the capitation arrangement?

It seems that if you had the negotiation arrangement separated—then how would you associate it with being part of the capitation arrangement?
***@***.***D71F41.F27E0780]



From: scott haselton ***@***.***>
Sent: Monday, March 22, 2021 3:55 PM
To: CMSgov/price-transparency-guide ***@***.***>
Cc: Fowler, Tammy ***@***.***>; Author ***@***.***>
Subject: [EXT] Re: [CMSgov/price-transparency-guide] Capitation Arrangements; How should partially capitated arrangements be notated? (#10)


Would you be able to provide a little more detail? For the partial capitation, would the non-capitation portion be something other than a fee-for-service or bundle arrangement?

For multiple arrangements in a single plan, you can add an additional object in the In Network Object in the in_network array<https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#in-network-object> to report of different types of arrangements.

For your example, would it be possible to cover a partial capitation with a fee-for-service component like this?:

{

  "in_network": [{

    "negotiation_arrangement": "capitation",

    "name": "Primary Care Capitation",

    "description": "Typical items and services for a primary care provider",

    "negotiated_rates": [{

      "negotiated_rate": 1400.00,

      "providers": [{

        "tin": "",

        "service_code": "01",

        "npi": ["6666666666", "7777777777", "8888888888", "9999999999"]

      },

    "covered_services": [{

      "billing_code_type": "CPT",

      "billing_code_type_version": "2020",

      "billing_code": "27447",

      "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee Joint"

    }]

  },{

    "negotiation_arrangement": "fee for service",

    "name": "Knee Replacement",

    "billing_code_type": "CPT",

    "billing_code_type_version": "2020",

    "billing_code": "27447",

    "description": "Arthroplasty, knee condyle and plateau, medial and lateral compartments",

    "negotiated_rates": [{

      "providers": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],

      "tin": "11-1111111",

      "negotiated_price": {

        "negotiated_type": "negotiated",

        "negotiated_rate": 123.45,

        "expiration_date": "2022-01-01"

      }

    }]

**Reply by shaselton-usds** [reply]
> It would seem that for partial capitation you would have negotiated_rates and have covered_services for each item/service that was part of the capitation arrangement?

I am curious if your situation for a partial capitation can't be represented in the example provided or if this is more of a question on what are all the values for the data element `negotiation_arrangement`?

**Comment by tdfow** [comment]
The rule indicates to see the IG for how to denote a partial capitation, so trying to understand how it would be noted.   In the above example, if the same billing code is reported under two different payment arrangements, where one is capitation and another FFS arrangement---is that how partial capitation would then be represented and identified?    Whereas, if the billing code value is ONLY under a capitation arrangement, covered_services then it would not be considered partial?

**Reply by tdfow** [reply]
If it was a partial capitation scenario, wouldn't the providers be the same --if you are reporting under both ffs and capitation ?   In your example, the providers are different--which would imply those are two different arrangements and not really a partial capitation scenario, correct?

**Reply by shaselton-usds** [reply]
The example definitely had dummy data.

You bring up a great point in linking the arrangements. The requirements to collect the same information (nothing additional) would still be the same, of course, but the arrangement could vary. 

You could include the FFS negotiated rate in capitation arrangement with adding a [negotiated_price](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-price-object) object in the [covered services object](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#covered-services-object) that is in the `covered_services` array. It would reduce NPI, TIN, etc redundancy as you've noted! It would be an optional field.

```
{
  "in_network": [{ 
    "negotiation_arrangement": "capitation",
    "name": "Primary Care Capitation",
    "description": "Typical items and services for a primary care provider",
    "negotiated_rates": [{
      "negotiated_rate": 1400.00,
      "providers": [{
        "tin": "",
        "service_code": "01",
        "npi": ["6666666666", "7777777777", "8888888888", "9999999999"]
      },
    "covered_services": [{
      "billing_code_type": "CPT",
      "billing_code_type_version": "2020",
      "billing_code": "27447",
      "description": "Under Repair, Revision, and/or Reconstruction Procedures on the Femur (Thigh Region) and Knee Joint",
      "negotiated_price": {
          "negotiated_type": "negotiated",
          "negotiated_rate": 123.45,
          "expiration_date": "2022-01-01"
      }
    }]
  }
```
This has the assumption that there is a single FFS negotiated price for the NPI and the capitation rate that it is paired with for the item/service.

How many plans are you working with that have these arrangements? Pretty esoteric or common?

**Reply by tdfow** [reply]
I will clarify, but I feel this is a common arrangement for some services, where there may be a FFS and capitation, where one would produce a claim and an encounter claim.     For the example above--you have reference to npi---yet, I am not seeing this in the schema---so is there a different schema for capitation or should the NPI values be reflected in 'providers'?

**Reply by tdfow** [reply]
@shaselton Is the direction for partial capitation scenarios to add a negotiated price object within the capitation for partial capitation type arrangements or would the same provider have the same items reported under FFS arrangement and the capitation under a capitation arrangement?     
Identifying the partial capitation scenario within the covered_services object would avoid the repeating of the provider information.  So, if this approach is taken, would the negotiated price object then only be added to the items/services that are both capitation and ffs and the capitation related item/services are shown only with the covered_services object without the negotiated price object?   If this is the direction, would the in-network  schema be updated for the covered_services object something like this?
  "covered_services": {
      "type": "object",
      "properties": {
        "billing_code_type": { "type": "string" },
        "billing_code_type_version": { "type": "string" },
        "billing_code": { "type": "string" },
        "description": { "type": "string" },
        "negotiated_price": {
          "$ref": "#/definitions/negotiated_price"
        }
      },
      "required": ["billing_code_type", "billing_code_type_version", "billing_code"]
    },

**Comment by tdfow** [comment]
#21  - related topic

**Reply by ks2178o2** [reply]
Was this resolved? I'm having a hard time pinning down the final solution to this scenario.
